src/hotspot/share/runtime/mutexLocker.hpp

```diff
@@ -172,7 +172,8 @@ extern Mutex* tty_lock;                          // lock to synchronize output.
 // Print all mutexes/monitors that are currently owned by a thread; called
 // by fatal error handler.
 void print_owned_locks_on_error(outputStream* st);
-
+// Unlock all Mutex/Monitors currently owned by a JavaThread.
+void unlock_locks_on_error(JavaThread* t);
```

Comment:
Nit: s/t/current/ as we must be applying this to the current thread.

Also as a matter of style the parameter names in the declaration should match those in the definition.